.gitmodules

```diff
@@ -15,3 +15,6 @@
 	path = lib/stm32lib
 	url = https://github.com/micropython/stm32lib
 	branch = work-F4-1.13.1+F7-1.5.0+L4-1.3.0
+[submodule "lib/littlefs"]
+	path = lib/littlefs
+	url = https://github.com/geky/littlefs.git
```

Comment:
When I go to https://github.com/geky/littlefs in my web browser it automatically redirects to https://github.com/ARMmbed/littlefs ?

Reply:
After I made this PR the littlefs project moved to the ARMmbed organisation.  I'll update the link.